### PR TITLE
Incorporate feedback

### PR DESCRIPTION
**Pull request by surma**
Got some feedback from @cwilso. This should address most of it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +201,7 @@ This is a very early stage proposal, so it has a few problems that we'll need to
 
 ### Returning references
 
-It will undoubtedly be useful to return instances of objects created in the tasklet to the main thread. The
+It will undoubtedly be undoubtedly useful to return instances of objects created in the tasklet. The
```
Comment (bfgeek) [inline README.md]: "undoubtedly be undoubtedly useful" 🤔

File: README.md [hunk 2]
```diff
@@ -220,32 +227,30 @@ export class CalendarEntry {
...
-// deceptively expensive.
-events.map(event => myCalender.generateShareLink(event.id));
-// Preferable:
-// myCalender.generateShareLinks(events);
+const events = await myCalendar.nextEvents()
+events.map(event => myCalender.generateShareLink(event.id)); // !!!
 ```
 
-### What gets exported?
+The last line is potentially problematic. `event.id` has been promisified. This line would create a lot of message passing under the hood: Every invocation of the `map()` callback would have to wait for `event.id` to resolve just pass a message back to the taskelt to invoke `generateShareLink`.
```
Comment (bfgeek) [inline README.md]: spelling taskelt

File: README.md (hunk 1 above)
Comment (surma) [inline README.md]: Uhm...

Comment (samdutton) [inline README.md]: Oops.

**Pull request merged**